src/Dapr.Client/DaprClientGrpc.cs

```diff
@@ -202,6 +202,36 @@ private static T ConvertFromInvokeBindingResponse<T>(Autogenerated.InvokeBinding
             return response.Data.Value.IsEmpty ? default : TypeConverters.FromAny<TResponse>(response.Data, this.jsonSerializerOptions);
         }
 
+        public override async ValueTask<IEnumerable<GetBulkItem>> GetBulkStateAsync(string storeName, IEnumerable<string> keys, int parallelism = -1, CancellationToken cancellationToken = default)
+        {
+            ArgumentVerifier.ThrowIfNullOrEmpty(storeName, nameof(storeName));
+            if (keys.Count() == 0)
+                throw new ArgumentException("keys do not contain any elements");
+
+             var getBulkStateEnvelope = new Autogenerated.GetBulkStateRequest()
+            {
+                StoreName = storeName,
+                Parallelism = parallelism
+            };
+            foreach (var key in keys)
+            {
+                getBulkStateEnvelope.Keys.Add(key);
```

Comment:
This is generated code so I can't really see it - does the type of `Keys` have an `AddRange` method? It's not just cleaner code - `AddRange` is a lot more efficient in common cases

Reply:
Uses `AddRange` now